### PR TITLE
make "See also" sections uniform

### PR DESCRIPTION
**Pull request by a14n**
The sections have now a uniform style as described in [Flutter Style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments).

I also ensured that lists have the same indentations.

### CODE REVIEW

**Comment by a14n** [comment]
Some comments/questions on what I met:

- when the element in list is only a link, there are lines with period and lines without period. Should we format all those lines the same way (with or without period?)
- some links are `<httpLink>` and other `[httpLink](httpLink)`. Is there a rule? If not should we only use `[httpLink](httpLink)`?
- most of the list elements start with a link. Should we change the lines that don't start with a link to enforce this rule?

File: packages/flutter/lib/src/material/icons.dart [hunk 1]
```diff
@@ -25,7 +25,7 @@ import 'package:flutter/widgets.dart';
 ///
 ///  * [Icon]
 ///  * [IconButton]
-///  * [design.google.com/icons](https://design.google.com/icons/)
+///  * <https://design.google.com/icons/>
```
Comment (Hixie) [inline packages/flutter/lib/src/material/icons.dart]: this (angle brackets, no trailing period, if the url is alone) is my preferred style for urls in these lists

**Comment by Hixie** [comment]
LGTM

**Comment by Hixie** [comment]
> Some comments/questions on what I met:
> 
> * when the element in list is only a link, there are lines with period and lines without period. Should we format all those lines the same way (with or without period?)
> * some links are `<httpLink>` and other `[httpLink](httpLink)`. Is there a rule? If not should we only use `[httpLink](httpLink)`?

If it's only a link,  then `<foo>`, no period.


> * most of the list elements start with a link. Should we change the lines that don't start with a link to enforce this rule?

Not necessarily, unless they're egregiously different for no reason. But sometimes it's just clearer to explain something before the link, or whatever.

**Pull request merged**